src/r2mm/profiles/ProfilesClient.ts

```diff
@@ -0,0 +1,34 @@
+import Axios, { AxiosResponse } from 'axios';
+import R2Error from 'src/model/errors/R2Error';
+
+export default class ProfilesClient {

```

Comment:
There's no need to use a class here since the file itself is already handling the encapsulation. Classes can be instantiated which doesn't really make sense here.

Also generally speaking, default exports are worse than named exports, so use them instead unless default exports are explicitly required (like with NextJS)

Reply:
The general pattern so far has been to use default exports as most things are classes. Named exports tend to be type declarations.

Reasoning is that it's easier to remember a class name than it is to remember a method name which ends up being more productive.